content-src/experiments/min-vid.yaml

```diff
@@ -97,5 +97,5 @@ contributors:
     display_name: 'Noitidart'
     title: 'Engineering Contributor'
     avatar: /static/images/experiments/min-vid/avatars/noitidart.jpg
-created: '2016-09-22T00:07:28.847430Z'
-order: 4
+    created: '2016-09-22T00:07:28.847430Z'
```

Comment:
Looks like accidental indentation? These last two properties should be part of the experiment overall, not the contributor